File: src/book-web/app/module/protocol/Item.go
package protocol

// Item 返回的章節信息
type Item struct {
	// 是否命中 緩存
	Hit bool
	// 當 緩存 失效時 返回的 新值
	Val string
	// 當 緩存 失效時 返回的 新增的 md5
	MD5 string
}

File: src/view/src/app/core/cache/item.ts
export class Item {
    // 是否命中 緩存
    Hit: boolean
    // 當 緩存 失效時 返回的 新值
    Val: string
    // 當 緩存 失效時 返回的 新增的 md5
    MD5: string

    ID?:string
}
